index.js

```diff
@@ -2015,6 +2015,23 @@
     return new RegExp(source, flags);
   });
 
+  //# regexEscape :: String -> String
+  //.
+  //. Takes a string which may contain regular expression metacharacters,
+  //. and returns a string with those metacharacters escaped.
+  //.
+  //. Properties:
+  //.
+  //.   - ∀ `s` ∈ String: `S.test(S.regex('', S.regexEscape(s)), s)`
```

Comment:
The notation makes sense! I agree with @scott-christopher in that the plaintext version is a bit more accessible. :+1: